src/app/organizations/collection/collection.component.scss

```diff
@@ -1,4 +1,40 @@
 .orgLogo {
-  max-height: 150px;
+  max-height: 100px;
   max-width: 200px;
 }
+
+.view-button {
+  background: #397a84;
+  color: white;
+  font-size: 12px;
+  padding: 0.5rem 1rem;
+}
+
+hr {
+  margin: 0.5rem 0;
+  border: 1px solid #dfe0e2;
+}
+
+h5 {
+  color: #151c3d;
```

Comment:
should make a style in the main scss. i feel like this color is going to be repeated a lot